### DISCUSSION TITLE
Same resource, but requires two routes

### DISCUSSION THREAD

**Discussion by dogeow** [filamentphp/filament#8267] [Help] [answered]
I have a Goods model, but my boss needs two lists, one to show the list of products under review, and one to show the list of products that have been approved. I made it, but their routes are the same, both are /admin/goods.

<img width="358" alt="image" src="https://github.com/filamentphp/filament/assets/7204604/d991685a-a46f-4e27-ba5e-0a240bd899ca">

**Comment by leandrocfe** [comment]
[customizing-the-page-url](https://filamentphp.com/docs/3.x/panels/pages#customizing-the-page-url) ?

**Reply by dogeow** [reply]
Thanks.
Routing is now ok, but the content of the main area is the same, such as: breadcrumbs, action, data. 
However, the code is different, getEloquentQuery, Action, $breadcrumb, etc.

**Reply by danharrin** [reply]
$modelLabel ?

**Reply by leandrocfe** [reply, marked as answer]
Not sure if this is the correct approach, but to avoid conflicts, I recommend duplicating your model. In a similar scenario, consider having a model called "Post," a resource named "PostResource," and if you need to create a "PublishedPost:

```php
//app/Models/PublishedPost.php
class PublishedPost extends Post
{
    protected $table = 'posts';

    protected static function booted(): void
    {
        static::addGlobalScope('published', function (Builder $builder) {
            $builder->wherePublished(true);
        });
    }
}
```

You can generate the new resource
`php artisan make:filament-resource PublishedPost`

**Reply by leandrocfe** [reply]
This is my opinion, but as Dan suggested, you could consider modifying other resource attributes, among other changes...

**Reply by dogeow** [reply]
My problem, I just copied the resource php file, I didn't copy the resource file or use `php artisan make:filament-resource`.
I haven't read too many documents, and every time I rush to you to ask some simple questions, I will study hard next.
